com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/SyncFlowableStep.java

```diff
@@ -45,6 +45,7 @@
     @Override
     public void execute(DelegateExecution context) {
         initializeStepLogger(context);
+        stepLogger.logFlowableTask();
```

Comment:
This should not be necessary. It's done in the preExecuteStep() method of StepHelper.